tljh-voila-gallery/tljh_voila_gallery/systemd-units/tljh-voila-gallery-builder.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description="Build the Docker images for the Voila Gallery Examples"
+
+[Timer]
+OnBootSec=1min
+OnUnitActiveSec=12h
```

Comment:
Let's bring this down to 2h maybe? 12h seems like a long time :D

Reply:
ah good catch, I thought it was already lowered. Should be fixed now.